basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
+
+echo $PATH_TO_FILES $PATH_TO_SHARD
+
+basin-generator $PATH_TO_FILES $PATH_TO_JSON
+basin $PATH_TO_JSON $PATH_TO_SHARD/output.shard
+eminem regenerate $PATH_TO_SHARD
```

Comment:
This is more a problem with eminem, but if manifest.json is a symlink to a manifest.json from a system-installed flatpak, then this will fail.